lore/metadata/__init__.py

```diff
@@ -243,11 +243,8 @@ class Fitting(Crud, Base):
     created_at = Column(DateTime, nullable=False, default=datetime.datetime.now)
     completed_at = Column(DateTime)
     snapshot_id = Column(Integer, ForeignKey('snapshots.id'), nullable=False, index=True)
-    train = Column(Float)
-    validate = Column(Float)
-    test = Column(Float)
+    scoring_metric = Column(String)
     score = Column(Float)
-    iterations = Column(Integer)
```

Comment:
`iterations` can be useful to detect some bogus data causing way early stopping.

Reply:
We still store iterations. But inside the `stats` column. Basically, all fit related data goes in there. This makes more sense to me since different estimators may store different fit related stats.